easy-digital-downloads.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Plugin Name: Easy Digital Downloads
+ * Plugin Name: Easy Digital Downloads - change here.
```

Comment:
Might want to remove this change. Looks like testing?

Just remove the changes, and commit then push. The PR will be updated.